test/integration/setup.js

```diff
@@ -138,23 +138,46 @@ const bundleTapeStandalone = async (destDir) => {
 
   const stream = b.bundle();
   const onceStreamEnd = awaitStreamEnd(stream);
-  stream.pipe(fs.createWriteStream(bundleFileName));
+  const destFileStream = fs.createWriteStream(bundleFileName);
+  const onceWritten = new Promise(resolve => {
+    destFileStream.on("close", resolve);
+  });
+  stream.pipe(destFileStream);
 
-  await onceStreamEnd;
+  await Promise.all([onceStreamEnd, onceWritten]);
 };
 
 test.onFailure(() => {
   process.exit(1);
 });
 
-const defineExtensionTests = ({description, extensions}) => {
+/**
+ * @param {object} parameters
+ * @param {string} parameters.description
+ * @param {string[]} parameters.extensions
+ * @param {boolean|string|string[]} [parameters.skip]
+ */
+const defineExtensionTests = ({description, extensions, skip}) => {
   for (const extensionDirName of extensions) {
     test(`${description} (test extension: ${extensionDirName})`, async (tt) => {
       let timeout;
       let driver;
       let server;
       let tempDir;
 
+      const browser = process.env.TEST_BROWSER_TYPE;
+
+      if (skip) {
```

Comment:
By removing from #163 the pieces related to Edge, there is no use of the skip property, and so we may choose to defer it or remove it as @Rob--W suggested in #163.